_config.yml

```diff
@@ -23,5 +23,7 @@ navigation:
   url: documentation.html
 - text: Community
   url: contribute.html
+- text: Blog
+  url: blog.html
```

Comment:
Something like https://github.com/domcleal/theforeman.org/commit/4c31f31653eccfad73a6132b25739a0dabdc53d6 would do it.